guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -1,107 +1,101 @@
 [id="Importing_Kickstart_Repositories_{context}"]
-= Importing Kickstart Repositories for {RHEL}{nbsp}{rhel-version}
+= Importing Kickstart Repositories for {RHEL}{nbsp}{el-major-version}
 
-To import Kickstart repositories for {RHEL}{nbsp}{rhel-version}, complete the following steps on {Project}.
+Use this procedure to import Kickstart repositories for {RHEL}{nbsp}{el-major-version}.
 
 .Procedure
 . Navigate to the Red{nbsp}Hat Customer Portal at https://access.redhat.com/[access.redhat.com] and log in.
-. In the upper left of the window, click *Downloads*.
-
-ifeval::["{rhel-version}" == "8"]
-. Click  {RHEL} 8*.
-. In the Download {RHEL} window, locate the binary DVD version of the ISO image, for example, *{RHEL} 8.1 Binary DVD*, and click *Download Now*.
-endif::[]
-
-ifeval::["{rhel-version}" == "7"]
-. To the right of *{RHEL} 7*, click *Versions 7 and below*.
-. From the *Version* list, select the required version of the {RHEL} 7, for example 7.7.
-. In the Download {RHEL} window, locate the binary DVD version of the ISO image, for example, *{RHEL} 7.7 Binary DVD*, and click *Download Now*.
-endif::[]
-
+. In the upper left corner of the window, click *Downloads*.
+. Click *{RHEL}*.
+. In the *Download {RHEL}* window, select a product variant and a product version from the list.
+For example, product variant *{RHEL} for x86_64* and product version *{el-major-version}.{el-minor-version}*.
+. Click *Switch to version 7 and below* above the *Product Variant* list.
+. Locate the full installation image, for example, *{RHEL} {el-major-version}.{el-minor-version} Binary DVD*, and click *Download Now*.
+. From the *Version* list, select the required version of the {RHEL}.
+. Locate the binary DVD version of the ISO image, for example, *{RHEL} Binary DVD*, and click *Download Now*.
 . When the download completes, copy the ISO image to {ProjectServer}.
 . On {ProjectServer}, create a mount point and temporarily mount the ISO image at that location:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# mkdir /mnt/_iso_
-# mount -o loop _rhel-binary-dvd.iso_ /mnt/_iso_
+# mkdir /mnt/iso
+# mount -o loop _rhel-binary-dvd.iso_ /mnt/iso
 ----
-
-ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
 +
+Replace _rhel-binary-dvd.iso_ with the name of your ISO image.
```

Comment:
```suggestion
Replace _<rhel-binary-dvd.iso>_ with the name of your ISO image.
```
Optional, but I think referring to replaceable that way identifies the replaceable better. On the other hand, I respect if the convention is to only use italic to identify replaceables. If you decide to go with  `<>`, please apply to entire file.

Reply:
No change. We generally use italics instead of `<REPLACE ME>`.